lib/iupac.rb

```diff
@@ -3,22 +3,26 @@
 module Iupac_convention
 
   Suffix =                    #   This is a constant definition
-      {                     
+      {                       # This is a hash (i.e. associative array  where a number of 'keys' are associated with corresponding 'values')
     :alkane => "ane",
      :alcohol => "ol",
      
   }
   
   
```

Comment:
But I want to put it into another variable.


On Tue, Mar 17, 2015 at 4:04 PM, awayanyway <notifications@github.com>
wrote:

> In lib/iupac.rb
> <https://github.com/nicolejung/IUPAC/pull/2#discussion_r26580279>:
>
> > @@ -3,22 +3,26 @@
> >  module Iupac_convention
> >
> >    Suffix =                    #   This is a constant definition
> > -      {
> > +      {                       # This is a hash (i.e. associative array  where a number of 'keys' are associated with corresponding 'values')
> >      :alkane => "ane",
> >       :alcohol => "ol",
> >
> >    }
> >
> >
>
> "1_2_3__abc".split(/_/) =>["1","2","3","","abc"]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nicolejung/IUPAC/pull/2/files#r26580279>.
>